src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -1840,6 +1840,29 @@ class Program
         End Function
 
         <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
+        <WorkItem(12254, "https://github.com/dotnet/roslyn/issues/12254")>
+        Public Async Function TestGenericCallOnTypeContainingAnonymousType() As Task
+            Using state = TestState.CreateCSharpTestState(
```

Comment:
Could this be a a test in `SymbolCompletionProviderTests` using http://source.roslyn.io/#Roslyn.Services.Editor.UnitTests/Completion/AbstractCompletionProviderTests.cs,3eeb03430e2433cc,references ?